cluster-api/src/main/java/io/aiven/klaw/clusterapi/services/SchemaService.java

```diff
@@ -469,6 +502,23 @@ public ApiResponse deleteSchema(ClusterTopicRequest clusterTopicRequest) {
               request,
               new ParameterizedTypeReference<>() {});
       log.info("Schema deleted {}", clusterTopicRequest);
+
+      try {
+        CompletableFuture.runAsync(
+                () -> {
+                  loadAllSchemasInfoFromCluster(
+                      clusterTopicRequest.getSchemaEnv(),
+                      clusterTopicRequest.getSchemaEnvProtocol(),
+                      clusterTopicRequest.getSchemaClusterIdentification(),
+                      true,
+                      SchemaCacheUpdateType.DELETE,
+                      clusterTopicRequest.getTopicName());
+                })
```

Comment:
updateSchemaCache can be used here if we pass in the SchemaCacheUpdateType

Reply:
there is one difference. its clusterTopicRequest and in other side its clusterSchemaRequest. However, I have extracted into a common method, which is better.